src/UXClient/Models/LineChartData.ts

```diff
@@ -3,7 +3,15 @@ import {ChartComponentData} from "./ChartComponentData";
 
 class LineChartData extends ChartComponentData {
     public timeMap: any = {};
+    get yExtents(): {}{
+        return this._yExtents;
```

Comment:
what happens if, on a subsequent render, there are less aggregates than the prior render? Will there be excess yExtents?

Reply:
might need more logic here to ensure there isn't stale extents here

Reply:
Added resetYExtents(), called before each render